_assets/stylesheets/pages/_app.scss

```diff
@@ -0,0 +1,220 @@
+    min-height: 44vh;
+  }
+  
+  .hero-transform {
+    margin-top: -140px;
+  }
+  .jumbotron .font-family-base-light {
+    font-size: 1.95rem !important;
+  }
+  
+  .font-family-condensed {
+    font-size: .7rem !important;
+  }
+ 
+  .privacy-format {
+    margin-right: auto;
+  }
+}
+
+/* Galaxy S5 & iPhone 6/7/8/Plus & Pixel 2 */
+@media (max-height: 736px) {
+.jumbo-viewport {
+  min-height: 57vh;
+}
+.hero-transform {
+  margin-top: -155px;
+  }
+}
+
+/* iPhone 4 & 5 */
+@media (max-height: 568px) {
+.jumbo-viewport {
+  min-height: 71vh;
+}
+.hero-transform {
+  margin-top: -110px;
+  }
+}
+
+/* iPad, iPad Pro, Laptop & Desktop */
+@media (min-width: 768px ) {
+  .jumbo-viewport {
+    min-height: 74vh;
+  }
+  .hero-transform {
+    margin-top: -34vh;
+  }
+  .jumbotron .font-family-base-light {
+    font-size: 4rem !important;
+  }
+    
+  .font-family-condensed {
+    font-size: 1rem !important;
+  }
+  .logo-margin{
+    margin-left: 30px;
+  }
+  
+  .parent {
+    display: flex !important;
+  }
+  
+.crds-shared-header .navbar-brand.logo {
+    left: auto !important;
+    position: static !important;
+    transform: none !important;
+    margin-top: 4px !important;
+    padding-top: 0;
+  }
+
+  .privacy-format {
+    margin-right: 30px;
+  }
+}  
+
+/* Landscape for mobile */
+@media (max-height: 414px) and (max-width: 823px) {
+  .jumbo-viewport {
+    min-height: 85vh;
+  }
+  .hero-transform {
+    margin-top: -80px;
+    }
+  }
+
+  @media (max-width: 767px) {
+    .hide-buttons {
+      visibility: hidden;
+    }
+  }
+/* Jumbotron Styling */
+img.jumbotron {
+  display: block;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+img.left {
+  display: block;
+  margin-right: auto;
+}
+
+img.right {
+  display: block;
+  margin-left: auto;
+}
+
+.jumbotron {
+  padding-top: 0px;
+}
+    
+.flex-header {
+  display: flex;
+}
+
+.title-top {
+  padding-top: 65px;
+}
+
+.font-family-condensed {
+  font-size: 1rem;
+}
+
+/* App Icon */
+.rounded-border {
+  border-radius: 22.5%;
+  height: 50px; 
+  width: 50px;
+}
+
+/* App Store & Google Play Buttons */
+.app-button {
+  height: 40px;
+  width: 135px;
+  margin-top: 8.5px;
+}
+
+.app-store-buttons {
+  margin-left: auto;
+}
+
+.center-app {
+  display: flex;
+  justify-content: center;
+}
+
+.content-middle {
+  margin-top: auto;
+  margin-bottom: auto;
+}
+
+/* Footer Classes */
+.footer-padding {
+  padding: 50px 0px 150px 0px;
+}
+
+.footer-bg {
+  background-image: url('https://crds-media.imgix.net/54CyLHouK1UsYqdcoVgrxa/d6efa29a446e388f18786ecb33a70758/footer-img.jpg');
+  background-size: cover;
+  background-position: center;
+}
+
+.footer-text {
+  color: white; 
+  opacity: 0.6;
+}
+
+.footer-display {
+  display: inline-flex;
+  flex-direction: column;
+}
+
+.footer-buttons {
+  margin-top: 50px;
+}
+
+.footer-margin {
+  margin-bottom: 30px;
+}
+
+p a:hover {
+  color: white;
+  text-decoration: none !important;
+  opacity: 0.6;
+}
+
+.crds-shared-header p a {
```

Comment:
I would avoid this type of css. This isn't scoped just to this page and you might high other `a` tags in places you don't want.